### PR TITLE
v5.2 website changes

### PR DESCRIPTION
**Pull request by Pessimistress**
#### Change List
- New: *showcase* section
- Fix: IconLayer cannot click bug
- Fix: ScatterplotLayer color picker
- Fix: Sidebar styling on mobile
- Fix: Map size

### CODE REVIEW

File: website/contents/pages.js [hunk 1]
```diff
@@ -90,6 +90,10 @@ export const examplePages = generatePath([
           code: getCodeUrl('examples/scatterplot')
         }
       },
+      {
+        name: '3D Indoor Scan',
+        external: 'http://uber.github.io/deck.gl/examples/point-cloud-laz'
```
Comment (ibgreen) [inline website/contents/pages.js]: If external, move to showcases?

File: website/src/components/header.js [hunk 2]
```diff
@@ -40,6 +40,7 @@ export default class Header extends Component {
           </div>
           <div className="links">
             <Link activeClassName="active" to="examples">Examples</Link>
+            <Link activeClassName="active" to="showcase">Showcase</Link>
```
Comment (ibgreen) [inline website/src/components/header.js]: Showcases?

File: website/contents/pages.js (hunk 1 above)
Comment (Pessimistress) [reply website/contents/pages.js]: This is our base point cloud layer example. It should be integrated when we get around to it.

File: website/src/static/markdown/showcase.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      <a href="http://pessimistress.github.io/ascii/">
+        <img src="images/showcase-ascii.jpg" />
+      </a>
+    </div>
+  </div>
+  <div class="thumb-desc">
+    <div>
+      <h2>Ascii Video Player</h2>
+      <p>Video processing on the fly.</p>
+      <p class="credit">By <a href="http://vis.gl/about">Uber Visualization</a> | <a href="https://github.com/pessimistress/deck.gl/tree/master/examples/ascii">Source code</a></p>
```
Comment (gnavvy) [inline website/src/static/markdown/showcase.md]: broken link if we were not to merge it to master.
href="https://github.com/Pessimistress/deck.gl/tree/ascii/examples/ascii"

Comment (Pessimistress) [reply website/src/static/markdown/showcase.md]: I have fixed the cross-origin issue, will open a PR to merge this demo into the main repo.

**Pull request merged**